python/processors/Stop0l_postproc.py

```diff
@@ -14,27 +14,36 @@
 from PhysicsTools.NanoSUSYTools.modules.updateJetIDProducer import *
 from PhysicsTools.NanoSUSYTools.modules.PDFUncertaintyProducer import *
 from PhysicsTools.NanoAODTools.postprocessing.modules.common.puWeightProducer import *
+from PhysicsTools.NanoAODTools.postprocessing.modules.jme.jecUncertainties import jecUncertProducer
 
+# JEC files are those recomended here (as of Mar 1, 2019)
+# https://twiki.cern.ch/twiki/bin/view/CMS/JECDataMC#Recommended_for_MC
+# Actual text files are found here 
+# https://github.com/cms-jet/JECDatabase/tree/master/textFiles
 DataDepInputs = {
-    "2016" : { "pileup": "Cert271036_284044_23Sep2016ReReco_Collisions16.root"
-   },
-    "2017" : { "pileup": "Cert294927_306462_EOY2017ReReco_Collisions17.root"
-   },
-    "2018" : { "pileup": "Cert314472_325175_PromptReco_Collisions18.root"
-   }
+    "2016" : { "pileup": "Cert271036_284044_23Sep2016ReReco_Collisions16.root",
+               "JECU": "Summer16_07Aug2017_V11_MC"
```

Comment:
Where are the files stored?

Reply:
Oops, I forgot this pr.  Because of the hardcode in the jec module i have to put them in NanoAOD tools.